lib/Styles/_label.scss

```diff
@@ -1,3 +1,11 @@
+.accent {
+    color: #{'mix(@accent_color, @fg_color, 0.27)'};
```

Comment:
I don't think this will produce very good results for e.g. yellow. In libadwaita I had _much_ better results with oklch colorspace (tho that would require you to update gtk once 4.16 is out)

Reply:
Well, that or calculate it programmatically ofc, then you don't need to wait.